classroom.rb

```diff
@@ -14,24 +14,36 @@ def add_student
 end
 
 class Book
+  @@books = []
   def initialize(title, author)
     @title = title
     @author = author
+    @@books.push({ 'title' => @title, 'author' => @author })
   end
   attr_accessor :title, :author
 
   def add_rental(date)
     new.Rental(date)
   end
+
+  def list_of_books
+    @@books.each_with_index do |book, idx|
+      puts "#{idx}) Title: #{book['title']}, Author: #{book['author']}"
+    end
+  end
+
+  def books_array
+    @@books
+  end
 end
```

Comment:
- [ ] I strongly suggest that you refactor each class in this file to their respective files. For example, it is better to have separate files for each class unless the class is member of the another class and it should exists only there. Therefore, I expect `Book.rb`, `Rental.rb` files created separately.